### ISSUE TITLE
Add ShellCheckBear to .coafile

### ISSUE THREAD

**Issue by jayvdb** [coala/coala#4952]
We dont have an explicit coding standard for shell scripts.

I believe we've mostly been using two space indents, but find the most common between here and the coala-bears repo, and force it here.
(There will be a second issue for updating coala-bears.)

Try to avoid using the docker; install shellcheck directly using apt-get if possible.

**Comment by nalinbhardwaj** [comment]
Leaving preliminary data here, and enforcing two spaces(since that's most common). Formatted as file_url = < indent level >

```
https://github.com/coala/coala-bears/blob/4926dc06a94b83ec0cf446074b4c2277d4ebba20/.ci/env_variables.sh = 2
https://github.com/coala/coala-bears/blob/b2d8fa97a89a99a04aabd8fb2b8da8adca8f46ea/.ci/deps.apt.sh = 2
https://github.com/coala/coala-bears/blob/b2d8fa97a89a99a04aabd8fb2b8da8adca8f46ea/.ci/deps.java.sh = 2
https://github.com/coala/coala-bears/blob/b2d8fa97a89a99a04aabd8fb2b8da8adca8f46ea/.ci/deps.opam.sh = 2
https://github.com/coala/coala-bears/blob/d9789b200ff89318ac96ca901a665c28a3953257/.ci/tests.sh = 2
https://github.com/coala/coala-bears/blob/e44890958290deb79a90d1440543825a962aa3f9/.ci/deps.pip.sh = 2
https://github.com/coala/coala-bears/blob/65d63043ce61ac1637d9b7df9a9682eac1973eda/.ci/deps.sh = 2
https://github.com/coala/coala-bears/blob/1ea886ab9dac649179ab8d6bea490dcc9ad1dd1f/.ci/check_maintainership.sh = 4
https://github.com/coala/community/blob/b2b1a6680d1aa96bbcea45cb7d3f23ea5484ad62/.ci/deploy.sh = 4
https://github.com/coala/community/blob/6b69490819a3785502269c03e2c4a7ebc8572269/orgname.sh = 4
```

**Comment by sangamcse** [comment]
@jayvdb, If @nalinbhardwaj is not working on this issue, can I take this one?

**Comment by Makman2** [comment]
Reassigning due to inactivity.

**Comment by nalinbhardwaj** [comment]
@sangamcse : yes, feel free to take this one, sorry about squatting on it for a long while.

However IIRC this was dependent on #4967 getting merged into a release. It _does_ appear that [0.12.0.dev20180101025653](https://pypi.python.org/pypi/coala/0.12.0.dev20180101025653) has the features so you'll need to upgrade everything to dev.

Also, to ease your work a bit, here's an old list(accurate on 13-Dec-2017) of the different indents in coala related projects, in format `link = <count of indent per level>`, those without indents are omitted:
```
https://github.com/coala/coala-bears/blob/4926dc06a94b83ec0cf446074b4c2277d4ebba20/.ci/env_variables.sh = 2
https://github.com/coala/coala-bears/blob/b2d8fa97a89a99a04aabd8fb2b8da8adca8f46ea/.ci/deps.apt.sh = 2
https://github.com/coala/coala-bears/blob/b2d8fa97a89a99a04aabd8fb2b8da8adca8f46ea/.ci/deps.java.sh = 2
https://github.com/coala/coala-bears/blob/b2d8fa97a89a99a04aabd8fb2b8da8adca8f46ea/.ci/deps.opam.sh = 2
https://github.com/coala/coala-bears/blob/d9789b200ff89318ac96ca901a665c28a3953257/.ci/tests.sh = 2
https://github.com/coala/coala-bears/blob/e44890958290deb79a90d1440543825a962aa3f9/.ci/deps.pip.sh = 2
https://github.com/coala/coala-bears/blob/65d63043ce61ac1637d9b7df9a9682eac1973eda/.ci/deps.sh = 2
https://github.com/coala/coala/blob/48f784acf51b8d3ea23b3e12359435d108bbe15b/.misc/check_unsupported.sh = 2
https://github.com/coala/coala/blob/0a01ef75580a42aca8f8335a6aab9ec2b78c6344/.misc/deps.sh = 2
https://github.com/coala/coala-bears/blob/1ea886ab9dac649179ab8d6bea490dcc9ad1dd1f/.ci/check_maintainership.sh = 4
https://github.com/coala/community/blob/b2b1a6680d1aa96bbcea45cb7d3f23ea5484ad62/.ci/deploy.sh = 4
https://github.com/coala/community/blob/6b69490819a3785502269c03e2c4a7ebc8572269/orgname.sh = 4
https://github.com/coala/coala/blob/925fd8ec725f0d0471f0c224b60973dc77001849/.misc/deploy.pypi.sh = 4
https://github.com/coala/coala/blob/48f72a03aa3c31abc049f3f61c6336a34bae588b/.misc/check_maintainership.sh = 4
```

I believe I did have a script to scrape this, but can't find it anymore. Sorry.

**Comment by sangamcse** [comment]
@nalinbhardwaj, Thank you for helping me with this. :smile:
